easybuild/easyconfigs/m/mpath/mpath-1.1.3-GCCcore-12.3.0.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'PythonBundle'
+
+name = 'mpath'
+version = '1.1.3'
+
+homepage = 'https://pypi.org/project/mpath/'
+description = """For now it's quit simple and get_path_info()
+method returns information about given path. It can be either
+a directory or a file path."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+dependencies = [
+    ('Python', '3.11.3'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('mclass', '1.3.4', {'checksums': ['414f786287b80a80261125fc9ed844dbf9093907e96999a2f0391a7b32a1cb60']}),
+    ('mpath', '1.1.3', {'checksums': ['f8ecaf91bded57e12676d0373bf27acc154dea56c386f6ad05b476bc2c6ddc4f']}),
+]
```

small nitpick, it's easier to read (and more consistent with other easyconfigs) like this:
```suggestion
exts_list = [
    ('mclass', '1.3.4', {
        'checksums': ['414f786287b80a80261125fc9ed844dbf9093907e96999a2f0391a7b32a1cb60'],
    }),
    ('mpath', '1.1.3', {
        'checksums': ['f8ecaf91bded57e12676d0373bf27acc154dea56c386f6ad05b476bc2c6ddc4f']
    }),
]
```